src/forms/renderers/CatalogName/AutoComplete.tsx

```diff
@@ -0,0 +1,131 @@
+/*
+  The MIT License
+
+  Copyright (c) 2017-2020 EclipseSource Munich
+  https://github.com/eclipsesource/jsonforms
+
+  Permission is hereby granted, free of charge, to any person obtaining a copy
+  of this software and associated documentation files (the "Software"), to deal
+  in the Software without restriction, including without limitation the rights
+  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+  copies of the Software, and to permit persons to whom the Software is
+  furnished to do so, subject to the following conditions:
+
+  The above copyright notice and this permission notice shall be included in
+  all copies or substantial portions of the Software.
+
+  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+  THE SOFTWARE.
+*/
+import {
+    EnumCellProps,
+    EnumOption,
+    JsonSchema7,
+    WithClassname,
+} from '@jsonforms/core';
+import {
+    Autocomplete,
+    AutocompleteRenderOptionState,
+    FilterOptionsState,
+    Input,
+} from '@mui/material';
+import { throttle } from 'lodash';
+import merge from 'lodash/merge';
+import React, { ReactNode } from 'react';
+
+export interface WithOptionLabel {
+    renderOption?(
+        props: React.HTMLAttributes<HTMLLIElement>,
+        option: any,
+        state: AutocompleteRenderOptionState
+    ): ReactNode;
+    filterOptions?(
+        options: any[],
+        state: FilterOptionsState<any>
+    ): EnumOption[];
+}
+
+const generateOptionsArray = (rootSchema: JsonSchema7, path: string) => {
+    return (
+        rootSchema.properties?.[path].examples?.map((pathVal) => {
+            return {
+                const: pathVal,
+                label: pathVal,
+            };
+        }) ?? []
+    );
+};
```

Comment:
Examples is just a string array - so making a function to make an object array that works with MUI Auto Complete easily.